test/everypolitician/popolo/json_test.rb

```diff
@@ -1,16 +1,22 @@
 require 'test_helper'
 
 class Everypolitician::Popolo::JsonTest < Minitest::Test
-  def test_legislative_periods
-    popolo_json = Everypolitician::Popolo::JSON.new(
+  def popolo_json
+    @popolo_json ||= Everypolitician::Popolo::JSON.new(
       events: [
         { name: 'Election 1', classification: 'general election', start_date: '2014-01-01' },
         { name: 'Term 2', classification: 'legislative period', start_date: '2015-01-01' },
         { name: 'Term 1', classification: 'legislative period', start_date: '2010-01-01' },
       ],
     )
+  end
+
+  def test_legislative_periods_ignores_other_event_types
     assert_equal 2, popolo_json.legislative_periods.size
     assert_equal 2, popolo_json.terms.size
```

Comment:
Ha, I felt a bit uneasy when I was writing these tests, but couldn't work out why, this explains it! 👍